src/app/showcase/card-showcase/card-showcase.component.html

```diff
@@ -2,5 +2,7 @@
   <kirby-card-example></kirby-card-example>
   <kirby-code-viewer [html]="exampleHtml"></kirby-code-viewer>
   <h4>Properties:</h4>
-  <kirby-showcase-properties [properties]="properties"></kirby-showcase-properties> 
+  <kirby-showcase-properties [properties]="properties"></kirby-showcase-properties>
+  <br>
+  <p>Shadow on kirby-card will be automatically removed if there is a (select)="onCardSelected()"</p>
```

Comment:
Nope, should be the opposite :)
If you can click/tap a card it should be elevated (= shadow)